.babelrc

```diff
@@ -1,5 +1,17 @@
 {
   "presets": [
-    "@babel/preset-env"
-  ]
-}
\ No newline at end of file
+    "@babel/preset-env",
+  ],
+  "plugins": [
+    [
+      "@babel/plugin-transform-modules-umd",
+      {
+        "exactGlobals": true,
+        "globals": {
+          "rx4": "disposableComponent",
```

Comment:
In webpack, we created a global variable `window['disposable-component']` which i think is a bit awkward with the hyphen. So I changed it to camelCase. This is a breaking change (albeit small one since maybe no one actually depends on that global variable right now), so we would have to decide whether to publish a new major version, to not make this change, or to publish as a minor/patch and hope no one notices.

Reply:
Is there a use case where we'd want to use both?

Reply:
i don't know -- the global variable UMD usage was created just for demoing the library i think. No one actually uses it in prod that way afaik